net.veloren.airshipper.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: net.veloren.airshipper
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions : [
+  org.freedesktop.Sdk.Extension.rust-stable
+]
+command: airshipper
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --persist=.local/share/airshipper
```

Comment:
Ideally the app should respect `XDG_DATA_HOME` and store the data there instead.

Reply:
Removed `--persist=.local/share/airshipper` as suggested, the app does respect `XDG_DATA_HOME`.